packages/celotool/src/cmds/generate/faucet-address.ts

```diff
@@ -0,0 +1,18 @@
+/* tslint:disable no-console */
```

Comment:
Wanted to keep this as a separate command, as the address returned from the "faucet" Account Type in account-address.ts is no longer what we're using

Reply:
Sgtm, though we are still using the faucet account type right? Just for the public facing faucet, not for the internal faucet?

Reply:
Yeah, on second thought I think this will lead to more confusion, so removing the command for now